discount/src/main/java/com/kyriba/conference/discount/DiscountApplication.java

```diff
@@ -0,0 +1,26 @@
+package com.kyriba.conference.discount;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@SpringBootApplication
+@EnableSwagger2
+public class DiscountApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(DiscountApplication.class, args);
+	}
+
+	@Bean
+	public Docket discountApi() {
+		return new Docket(DocumentationType.SWAGGER_2)
+				.select()
+				.apis(RequestHandlerSelectors.basePackage("com.kyriba.conference.discount.api"))
```

Comment:
Add global API info and apply filter on URL level.

Reply:
done